indico/modules/categories/controllers/display.py

```diff
@@ -581,8 +590,12 @@ def _process(self):
                  .filter(Event.starts_between(self.start_dt, self.end_dt),
                          Event.is_visible_in(self.category.id),
                          ~Event.is_deleted)
-                 .options(load_only('id', 'title', 'start_dt', 'end_dt', 'category_id')))
+                 .options(load_only('id', 'title', 'start_dt', 'end_dt', 'category_id', 'own_venue_id')))
         events = self._get_event_data(query)
+        raw_categories = Category.query.options(load_only('id', 'title')).all()
```

Comment:
you can't do that ;) it's a huge list (over 10k in the CERN instance) which will cause a huge slowdown when processing it in the JS code!

however, using the `detailed_category_chain` (make sure to undefer it to avoid O(n) queries) of each event should allow you to get rid of this altogether

Reply:
Oh, didn't know that. Fixed in [0841c74](https://github.com/indico/indico/pull/6110/commits/0841c74514dfba918707224da94aa199e184a984).